docs/resources/cce_pvc.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# g42cloud_cce_pvc
+
+Manages a CCE Persistent Volume Claim resource within G42Cloud.
+
+## Example Usage
+
+### Create PVC with EVS
+
+```hcl
+variable "cluster_id" {}
+variable "namespace" {}
+variable "pvc_name" {}
+
+resource "g42cloud_cce_pvc" "test" {
+  cluster_id = var.cluster_id
+  namespace  = var.namespace
+  name       = var.pvc_name
+
+  annotations = {
+    "everest.io/disk-volume-type" = "SSD"
+  }
+
+  storage_class_name = "csi-disk"
+  access_modes       = ["ReadWriteOnce"]
+  storage            = "10Gi"
+}
+```
+
+### Create PVC with OBS
+
+```hcl
+variable "cluster_id" {}
+variable "namespace" {}
+variable "pvc_name" {}
+
+resource "g42cloud_cce_pvc" "test" {
+  cluster_id = var.cluster_id
+  namespace  = var.namespace
+  name       = var.pvc_name
+
+  annotations = {
+    "everest.io/obs-volume-type" = "STANDARD"
+    "csi.storage.k8s.io/fstype"  = "obsfs"
+  }
+
+  storage_class_name = "csi-obs"
+  access_modes       = ["ReadWriteMany"]
+  storage            = "1Gi"
+}
+```
+
+### Create PVC with SFS
+
+```hcl
+variable "cluster_id" {}
+variable "namespace" {}
+variable "pvc_name" {}
+
+resource "g42cloud_cce_pvc" "test" {
+  cluster_id         = var.cluster_id
+  namespace          = var.namespace
+  name               = var.pvc_name
+  storage_class_name = "csi-nas"
+  access_modes       = ["ReadWriteMany"]
+  storage            = "10Gi"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the PVC resource.
+  If omitted, the provider-level region will be used. Changing this will create a new PVC resource.
+
+* `cluster_id` - (Required, String, ForceNew) Specifies the cluster ID to which the CCE PVC belongs.
+  Changing this will create a new PVC resource.
+
+* `namespace` - (Required, String, ForceNew) Specifies the namespace to logically divide your containers into different
+  group. Changing this will create a new PVC resource.
+
+* `name` - (Required, String, ForceNew) Specifies the unique name of the PVC resource. This parameter can contain a
+  maximum of 63 characters, which may consist of lowercase letters, digits and hyphens (-), and must start and end with
+  lowercase letters and digits. Changing this will create a new PVC resource.
+
+* `annotations` - (Optional, Map, ForceNew) Specifies the unstructured key value map for external parameters.
+  Changing this will create a new PVC resource.
+
+* `labels` - (Optional, Map, ForceNew) Specifies the map of string keys and values for labels.
+  Changing this will create a new PVC resource.
+
+* `storage_class_name` - (Required, String, ForceNew) Specifies the type of the storage bound to the CCE PVC.
+  The valid values are as follows:
+    + **csi-disk**: EVS.
+    + **csi-obs**: OBS.
+
```

Comment:
```suggestion
* `storage_class_name` - (Required, String, ForceNew) Specifies the type of the storage bound to the CCE PVC.
  The valid values are as follows:
    + **csi-disk**: EVS.
    + **csi-obs**: OBS.
  Changing this will create a new PVC resource.
```